aip/general/0136.md

```diff
@@ -44,6 +44,9 @@ services. The bullets below apply in all three cases.
   - The name **must not** contain prepositions ("for", "with", etc.).
   - The verb in the name **should not** contain any of the standard method verbs ([Get][],
     [List][], [Create][], [Update][], [Delete][]).
+  - The name **must not** start or end with "async". The verb `Begin` **may**
```

Comment:
Nit: Since in the same bullet `Begin` gets formatting, should "async" get it too?
```suggestion
  - The name **must not** start or end with `Async`. The verb `Begin` **may**
```

Reply:
Done.